docs/proposals/PROPOSAL_E.md

```diff
@@ -80,17 +79,21 @@ Extend the Image Manifest with a refers field (existing registries should ignore
 {
   "schemaVersion": 2,
   "mediaType": "application/vnd.oci.image.manifest.v1+json",
```

Comment:
Should this be `application/vnd.oci.artifact.manifest.v1+json` ? Or is there a scenario where an image manifest can refer another image manifest?